src/components/albumMobile/albumMobile.scss

```diff
@@ -0,0 +1,61 @@
+.album-mobile__container {
+  background: linear-gradient(transparent 0, rgba(0, 0, 0, 0.7) 100%);
+  height: 100%;
+  position: absolute;
+  width: 100%;
+  display: flex;
+  gap: 10px;
+  padding: 30px 0 0 0;
+  flex-direction: column;
+
+  &-content {
+    text-align: center;
+
+    &-image {
+      width: 260px;
+    }
+  }
+
+  &-info {
```

Comment:
I think it makes sense, I was doing it like this at the beginning and I created a media query for mobile, but on mobile, it has a different content, it had information that was not necessary, so I used display: hidden, so I thought it would be bad when adding other content of a page that would be different, but will be other components, so I think it makes sense, I'll change

Reply:
I changed it to just one rai component